examples/containerlab/.topology.clab.yml.bak

```diff
@@ -0,0 +1,22 @@
+name: bonding
+mgmt:
+  network: inmanta_mgmt
+  ipv4_subnet: 10.0.0.0/24
+topology:
+  nodes:
+    R1:
```

Comment:
Very minor, but I would use another name than `R1`.  We usually use that to designate a router, this is rather a client, so maybe `C1`?